packages/next/src/build/create-compiler-aliases.ts

```diff
@@ -272,6 +273,7 @@ export function createRSCAliases(
       alias = Object.assign(alias, {
         'react/jsx-runtime$': `next/dist/server/future/route-modules/app-page/vendored/${layer}/react-jsx-runtime`,
         'react/jsx-dev-runtime$': `next/dist/server/future/route-modules/app-page/vendored/${layer}/react-jsx-dev-runtime`,
+        'react/compiler-runtime$': `next/dist/server/future/route-modules/app-page/vendored/${layer}/compiler-runtime`,
```

Comment:
i think this requires adding a new file in `packages/next/src/server/future/route-modules/app-page/vendored/{rsc,ssr}`, currently this won't resolve to anything

Reply:
oh good point, thanks